SampleUsages/TestScenario.cs

```diff
@@ -26,7 +27,7 @@ class TestScenario
         public int WarmUpTimeInMinutes { get; set; }
         private ILogger _logger { get; set; }
 
-        internal void RunScenario(ILogger logger)
+        internal void RunScenario(ILogger logger, ServerlessResultManager.TestScenario testScenario = null)
```

Comment:
I agree that's confusing. 
But for slightly different reason - argument is an DB object. I'll call it that way.